stdlib/public/Concurrency/Task.cpp

```diff
@@ -1594,6 +1594,15 @@ void (*swift::swift_task_asyncMainDrainQueue_hook)(
     swift_task_asyncMainDrainQueue_original original,
     swift_task_asyncMainDrainQueue_override compatOverride) = nullptr;
 
+SWIFT_CC(swift)
+void swift::swift_task_startOnMainActor(AsyncTask* task) {
```

Comment:
Should this function be overrideable? Seems very basic, not much need for it, but....

Reply:
I... maybe? I wasn't sure. What's your rule-of-thumb for overriding things? I think the flags are the part I'd be most worried about, but those are generated in Swift.

Reply:
Okay, it's overridable now.